docs/coins.md

```diff
@@ -1,63 +1,62 @@
 # Full list
 
-This list is generated from `src/coins.json`
+This list is generated from [./coins.json](../coins.json)
```

Comment:
This file is generated, so we need to update this file `codegen/lib/templates/coins.md.erb`